react/features/filmstrip/actions.web.js

```diff
@@ -80,8 +86,41 @@ export function setVerticalViewDimensions() {
     return (dispatch: Dispatch<any>, getState: Function) => {
         const state = getState();
         const { clientHeight = 0, clientWidth = 0 } = state['features/base/responsive-ui'];
+        const { width: filmstripWidth } = state['features/filmstrip'];
         const disableSelfView = shouldHideSelfView(state);
-        const thumbnails = calculateThumbnailSizeForVerticalView(clientWidth);
+        const resizableFilmstrip = isFilmstripResizable(state);
+        const thumbnails = resizableFilmstrip
+            ? calculateThumbnailSizeForResizableVerticalView(clientWidth, filmstripWidth.current ?? 0)
+            : calculateThumbnailSizeForVerticalView(clientWidth);
+
+        let gridView = {};
+
+        if (resizableFilmstrip && ((filmstripWidth.current ?? 0) > FILMSTRIP_GRID_BREAKPOINT)) {
```

Comment:
It looks to me that in this case we don't use calculated thumbnail size from `calculateThumbnailSizeForResizableVerticalView` except the width of the thumbnail in order to calculate the width of the `remoteVideos` div. Is this correct?

If this is the case can we do the whole thing a little bit more obvious what we actually use and calculate?

For example:
I see that the new height value in `dimensions.gridView.filmstripHeight` is actually used for the same element where we use `dimensions.remoteVideosContainer.height`. Maybe we can store the `dimensions.gridView.filmstripHeight` in `dimensions.remoteVideosContainer.height`?
Also in the case where we have grid I don't see why we need to calculate the or pass `local` and `remote` width and height at all. Also in this case the calculation of the `remoteVideosContainer`'s height and width can be done outside `calculateThumbnailSizeForResizableVerticalView`